drv/psc-seq-server/src/main.rs

```diff
@@ -250,6 +259,18 @@ enum PresentState {
         // with a "sticky fault" situation.
         turn_on_deadline: u64,
     },
+
+    /// We are allowing the ON signal to float active, as in the `On` state, but
+    /// we're not convinced the PSU is okay. We enter this state when bringing a
+    /// PSU out of an observed fault state, and it causes us to ignore its OK
+    /// output for a brief period (the deadline parameter).
```

Comment:
very minor nit: maybe worth referencing the duration constant here?
```suggestion
    /// output for a brief period ([`PROBATION_MS`]), which is represented
    /// by the deadline parameter.
```
(or something)